frequency-chain/style-guide
# Grid component

# Problem
Add Grid component with breakpoints, columns, gutter & margins according to design specification.

Closes #3 

Copies method for sizing from [Frequency.xyz](https://frequency-chain/frequency-xyz)

## Steps to Verify:
`npm run storybook`
Go to Design System --> Grid and check out different sizes.
Compare with Figma design linked in Issue.

## Screenshots (optional):
![Grid](https://github.com/user-attachments/assets/b8e1edaf-3ca2-4f10-8938-1052efbc9981)
